src/store/modules/layers.store.js

```diff
@@ -367,20 +367,6 @@ const actions = {
             log.error('Failed to set layer year, invalid payload', layerId, year)
         }
     },
-    moveActiveLayerBack({ commit, state, getters }, layerId) {
-        const activeLayer = getters.getActiveLayerById(layerId)
-        if (activeLayer) {
-            // checking if the layer can be put one step back
-            const currentIndex = state.activeLayers.indexOf(activeLayer)
-            if (currentIndex > 0) {
-                commit('moveActiveLayerFromIndexToIndex', {
-                    layer: activeLayer,
-                    startingIndex: currentIndex,
-                    endingIndex: currentIndex - 1,
-                })
-            }
-        }
-    },
```

Comment:
?

Reply:
I must have removed it at the same time as the function I wanted to remove by mistake. I'm putting this one back